jupyterlab_bigquery/src/components/shared/info_card.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Paper } from '@material-ui/core';
+import { stylesheet } from 'typestyle';
+
+const localStyles = stylesheet({
+  container: {
+    display: 'flex',
+    alignItems: 'stretch',
+    marginBottom: '12px',
+    backgroundColor: 'var(--jp-layout-color0)',
+    justifyContent: 'space-between',
+    paddingRight: '12px',
+  },
+  leftSide: {
+    display: 'flex',
+  },
+  icon: {
+    marginRight: '12px',
+  },
+  messageSpace: {
+    padding: '4px 8px',
+    display: 'flex',
+    alignItems: 'center',
+  },
+});
+
+interface InfoCardProps {
+  message: React.ReactNode;
+  button?: React.ReactNode;
+  color: string;
+  // TODO: figure out type so only material ui icons accepted,
+  // and make it work with using .type
+  icon: any;
```

Comment:
I couldn't figure out how to type the passed in icon so that I would be able to use `icon.type`...also ideally I would type this so that only icon type components can be passed in, but I'm not sure if that's possible.

Reply:
Maybe you could make a placeholder type until then. At least other devs would know the purpose:

```
// TODO: figure out type so only material ui icons accepted,
// and make it work with using .type
type MaterialUIIcon = any;

interface InfoCardProps {
  message: React.ReactNode;
  button?: React.ReactNode;
  color: string;
  icon: MaterialUIIcon;
}
```

Reply:
Sounds good